.travis.yml

```diff
@@ -1,20 +1,21 @@
 sudo: false
 
+dist: xenial
 language: python
 
 python:
-  - 2.7
+  - 3.6
```

Comment:
Small note, but I think most of OpenEdX is still on 3.5. And this packages's readme only mentions 3.5.

Reply:
It looks like the rest of the config was already claiming 3.6, despite the README. I think that might call for a second PR—and I'm planning on putting one in to make this OEP-18 compliant, which might be a good time to make that change as well.